src/auth.ts

```diff
@@ -143,8 +146,10 @@ export function setupAuth(app: express.Application, userModel: UserModel) {
   const customTokenExtractor = function (req: express.Request) {
     let token: string | null = null;
     if (req?.cookies?.jwt) {
+      console.log("COOKIES", req.cookies);
```

Comment:
Do we want to keep these console logs here?

Reply:
Oh no, whoops!